packer/packer.json

```diff
@@ -47,6 +47,24 @@
         "kubernetes_version": "{{user `kubernetes_version`}}",
         "kubernetes_cni_version": "{{user `kubernetes_cni_version`}}"
       }
+    },
+    {
+      "name": "gcp-ubuntu-16.04",
```

Comment:
Is there a reason we are only adding Ubuntu? I think we will want to keep parity for CentOS & RHEL for every provider.